### PR TITLE
Added study intro scene

### PR DESCRIPTION
**Pull request by ColinT**

### CODE REVIEW

**Review by anson-li — APPROVED** [review]

**Comment by anson-li** [comment]
@ColinT , just a nitpick but is it possible to add a dev boolean to skip the cutscenes if needed? Just so we can iterate locally without being blocked by the intro.

**Comment by ColinT** [comment]
@anson-li Yes, I can add that before merging. I'll add a configuration autoload where you can select which scene to start with.

**Review by anson-li — APPROVED** [review]
LGTM

**Pull request merged**
